docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -1,63 +1,69 @@
 
-### Layout
+## Free tools
+
+### Admin frameworks
+
+- [React-admin](https://github.com/marmelab/react-admin): A front-end framework for building data-driven applications running in the browser on top of REST/GraphQL APIs.
+- [Refine](https://github.com/refinedev/refine): An open source, headless React meta-framework for the rapid development of web applications.
+
+### Theming tools
+
+- [Material palette generator](https://m2.material.io/inline-tools/color/): The official Material Design 2 palette generator tool.
+
+### Supplementary components
+
+#### Layout
 
 - [MUI Treasury Layout](https://mui-treasury.com/?path=/docs/layout-v6-introduction--docs): Components to handle the overall layout of a page. Check out examples such as [a legacy.reactjs.org clone](https://mui-treasury.com/?path=/story/layout-v6-app-react-legacy--react-legacy).
 
-### Image
+#### Image
 
-- [mui-image](https://github.com/benmneb/mui-image): The only Material UI image component to satisfy the Material Design guidelines for loading images.
-- [material-ui-image](https://mui.wertarbyte.com/#material-ui-image): Adds a "materializing" effect to images so they fade in like [Material Design's image loading pattern](https://m1.material.io/patterns/loading-images.html) suggests.
+- [mui-image](https://github.com/benmneb/mui-image): The only Material UI image component to satisfy the Material Design 2 guidelines for loading images.
 
-### Chips
+#### Chips input
 
 - [mui-chips-input](https://github.com/viclafouch/mui-chips-input): A chips input designed for use with Material UI.
 
-### Phone Number
+#### Phone number input
 
 - [mui-tel-input](https://github.com/viclafouch/mui-tel-input): A phone number input designed for use with Material UI, built with [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js).
 
-### One-Time Password
+#### One-time password input
 
-- [mui-otp-input](https://github.com/viclafouch/mui-otp-input): A One-Time Password input designed for use with Material UI.
+- [mui-otp-input](https://github.com/viclafouch/mui-otp-input): A one-time password input designed for use with Material UI.
 
-### File
+#### File input
 
 - [mui-file-input](https://github.com/viclafouch/mui-file-input): A file input designed for use with Material UI.
 
-### Color picker
+#### Color input
 
 - [mui-color-input](https://github.com/viclafouch/mui-color-input): A color input designed for use with Material UI, built with [TinyColor](https://tinycolor.vercel.app/).
-- [material-ui-color](https://github.com/mikbry/material-ui-color): Collections of color components for Material UI. No dependencies, small, highly customizable, and supports theming.
-
-### Rich text
-
-- [mui-tiptap](https://github.com/sjdemartini/mui-tiptap): A customizable Material UI-styled WYSIWYG (What You See Is What You Get) rich text editor, using [Tiptap](https://tiptap.dev/).
 
-### Sparkline
+#### Rich text editor
```

Comment:
Rich text -> Rich text editor

Yeah, I was just about to make this same change. I got confused reading Rich text.